### PR TITLE
Fix interface and proto definition

### PR DESCRIPTION
**Pull request by Hosshii**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

ref #3442, #3449

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/insight/insightstore/deploymentstore.go [hunk 1]
```diff
@@ -26,14 +26,14 @@ var (
 )
 
 type DeploymentStore interface {
-	// GetDeployments returns slice of Deployment sorted by startedAt ASC.
-	GetDeployments(ctx context.Context, projectID string, from, to int64, minimumVersion *model.InsightDeploymentVersion) ([]*model.InsightDeployment, error)
+	// List returns slice of Deployment sorted by startedAt ASC.
+	List(ctx context.Context, projectID string, from, to int64, minimumVersion *model.InsightDeploymentVersion) ([]*model.InsightDeployment, error)
 
 	PutDeployments(ctx context.Context, projectID string, deployments []*model.InsightDeployment, version *model.InsightDeploymentVersion) error
```
Comment (knanao) [inline pkg/insight/insightstore/deploymentstore.go:32]: imo, we can also fix it like this
```suggestion
	Put(ctx context.Context, projectID string, deployments []*model.InsightDeployment, version *model.InsightDeploymentVersion) error
```

**Comment by Hosshii** [comment]
Thank you!
I fixed. PTAL.

**Comment by knanao** [comment]
Thanks!
/lgtm

**Comment by khanhtc1202** [comment]
Here you go
/approve

**Pull request merged**